ComSemApp/student/views.py

```diff
@@ -756,10 +754,10 @@ def get_context_data(self, **kwargs) -> dict[str, Any]:
                 
                 # These weights are based on the formula: attempt - min / range
                 # the days_since_review score is subtracted from one since less days since review is higher familiarity
-                expression.norm_figs['correct_attempts'] = max(0, exp_data[expression]['correct_attempts'] - mins['correct_attempts']) / ranges['correct_attempts']
+                expression.norm_figs['correct_attempts'] = max(1, exp_data[expression]['correct_attempts'] - mins['correct_attempts']) / ranges['correct_attempts']
```

Comment:
I replaced these values with 1 because the previous algorithm would have given 0's if the min and max are equal